### PR TITLE
#616: [Style Linter] Add relative webpack loader (closes #616)

### LINKED ISSUE #616: [Style Linter] Add relative webpack loader
← #614

## requirements
- Make linting easy with [stylelint-webpack-loader](https://github.com/vieron/stylelint-webpack-plugin)

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by ascariandrea**
Issue #616

### tests performed
- Style lint warnings messages appear in `stdout` ✔️ 
- Fixing style resolve warnings ✔️ 
- No errors in CI ✔️

### CODE REVIEW

File: showroom/webpack.base.babel.js [hunk 1]
```diff
@@ -11,7 +12,12 @@ const paths = {
 module.exports = {
 
   plugins: [
-    new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /de|fr|en/)
+    new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /de|fr|en/),
+    new StyleLintPlugin({
+      files: '**/*.s?(a|c)ss',
```
Comment (FrancescoCioria) [inline showroom/webpack.base.babel.js]: as we're checking only `scss` syntax (`syntax: 'scss'`) couldn't this be simplified to `**./*.scss`?

File: src/Panel/panel.scss [hunk 2]
```diff
@@ -30,6 +30,7 @@ $content-title-color: $brc-darkGrey !default;
 
 .panel {
   position: relative;
+  top: 10;
```
Comment (FrancescoCioria) [inline src/Panel/panel.scss]: why this?

File: showroom/webpack.base.babel.js (hunk 1 above)
Comment (ascariandrea) [inline showroom/webpack.base.babel.js]: Yep and I did it!

File: src/Panel/panel.scss (hunk 2 above)
Comment (ascariandrea) [inline src/Panel/panel.scss]: A left over for testing purpose 🙈

**Pull request merged**